source/basic.tex

```diff
@@ -6374,31 +6374,42 @@
 variable template specialization has ordered initialization.
 \end{note}
 
+\pnum
+A declaration \tcode{D} is
+\defn{appearance-ordered} before a declaration \tcode{E} if
+\begin{itemize}
+\item \tcode{D} appears in the same translation unit as \tcode{E}, or
+\item the translation unit containing \tcode{E}
+has an interface dependency on the translation unit containing \tcode{D},
+\end{itemize}
+in either case prior to \tcode{E}.
```

Comment:
It's not clear what we're talking about in this last phrase, especially when applied to the 2nd bullet.  Can this be clarified?

Reply:
This is intentionally handwaving around our not having precise wording for this constraint; the same handwaving occurs in [module.reach]p1. Filed #3502 to track this.